named.go

```diff
@@ -157,6 +157,10 @@ func bindAnyArgs(names []string, arg interface{}, m *reflectx.Mapper) ([]interfa
 // type, given a list of names to pull out of the struct.  Used by public
 // BindStruct interface.
 func bindArgs(names []string, arg interface{}, m *reflectx.Mapper) ([]interface{}, error) {
+	if maparg, ok := arg.(map[string]interface{}); ok {
```

Comment:
`bindAnyArgs()` already has:

```
func bindAnyArgs(names []string, arg interface{}, m *reflectx.Mapper) ([]interface{}, error) {
    if maparg, ok := arg.(map[string]interface{}); ok {
        return bindMapArgs(names, maparg)
    }
    return bindArgs(names, arg, m)
}
```

Reply:
There's only one use of bindArgs, and it seems safe to replace it with `bindAnyArgs`, so doing that in a separate branch.